js/plugin/connection.ts

```diff
@@ -66,32 +86,38 @@ export const ConnectionMixin = (SuperClass) => {
       const conn = (await hass()).connection;
       this.connection = conn;
 
-      // Subscribe to configuration updates
-      conn.subscribeMessage((msg) => this.incoming_message(msg), {
-        type: "browser_mod/connect",
-        browserID: this.browserID,
-      });
+      const setupBrowserModConnection = () => {
+        this.LOG("Subscribing to browser_mod/connect events");
+        conn.subscribeMessage((msg) => this.incoming_message(msg), {
+          type: "browser_mod/connect",
+          browserID: this.browserID,
+        });
+      };
+
+      // Initial connect subscription
+      setupBrowserModConnection();
```

Comment:
This work exactly the same way it worked previously – subscription is made to existing WS connection (assuming HA is up-and-running).

Reply:
Now that there is code reuse, `setupBrowserModConnection` may better be `connectBrowsertModComponent` or similar. i.e. this is just the connection, not really any 'setup'.